.github/workflows/test.yml

```diff
@@ -97,19 +185,19 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - name: Create local cache directory for Docker images
-      if: ${{ matrix.os == 'ubuntu-latest' }}
+      if: ${{ matrix.os == 'ubuntu-20.04' }}
```

Comment:
I think this step should run if on both latest and 20.04, i.e.

```
if: ${{ matrix.os in ('ubuntu-latest', 'ubuntu-20.04') }}
```

Occurs multiple times.

Reply:
DONE